.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,32 +1,20 @@
 ---
 name: Bug report
-about: Report a bug encountered while operating KUDO
+about: Report a bug on the KUDO docs site
 title: ''
-labels: kind/bug
-assignees: ''
+labels: ''
+assignees: meichstedt
```

Comment:
🎉 welcome to the team. Are you going to be joining all the meetings? :D

Reply:
Yes!